From 85e0e51cc9af88615233c7b77c57053bb542b047 Mon Sep 17 00:00:00 2001
From: Richard Zhu <r65037@freescale.com>
Date: Thu, 9 Oct 2014 15:58:41 +0800
Subject: [PATCH 0485/1074] MLK-9671 pcie: PERST_GPIO and POWER_GPIO are
 currently swapped

Fix the GPIO assignments as per the board schematics.

Signed-off-by: Richard Zhu <richard.zhu@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dts |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 0024d1d..60fffbb 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -417,8 +417,8 @@
 };
 
 &pcie {
-	power-on-gpio = <&gpio2 0 0>;
-	reset-gpio = <&gpio2 1 0>;
+	power-on-gpio = <&gpio2 1 0>;
+	reset-gpio = <&gpio2 0 0>;
 	status = "okay";
 };
 
-- 
1.7.5.4

